host-controller/src/main/java/org/jboss/as/host/controller/mgmt/HostControllerRegistrationHandler.java

```diff
@@ -337,6 +339,7 @@ public void execute(final OperationContext context, final ModelNode operation) t
                 final int major = hostInfo.getManagementMajorVersion();
                 final int minor = hostInfo.getManagementMinorVersion();
                 final int micro = hostInfo.getManagementMicroVersion();
+                ModelVersion hostVersion = ModelVersion.create(major, minor, micro);
```

Comment:
Optional: It could be final ... just to follow the same pattern used in this code and to reflect the intention. I leave it up to your criteria.

Reply:
This article best summarizes my opinion on the use of final for local variables: https://brytecode.be/articles/use-of-the-final-keyword-in-java/#final-for-local-variables